interface/super/edit_layout_props.php

```diff
@@ -0,0 +1,185 @@
+<?php
+/**
+ * Edit Layout Properties.
+ *
+ * Copyright (C) 2016-2017 Rod Roark <rod@sunsetsystems.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Rod Roark <rod@sunsetsystems.com>
+ * @link    http://www.open-emr.org
+ */
+
+$sanitize_all_escapes  = true;
+$fake_register_globals = false;
+
+require_once("../globals.php");
+require_once("$srcdir/acl.inc");
+require_once("$srcdir/formdata.inc.php");
+require_once("$srcdir/htmlspecialchars.inc.php");
```

Comment:
remove the formdata.inc.php and htmlspecialchars.inc.php includes since these are brought in via the autoloader:
https://github.com/sunsetsystems/openemr/blob/c0f12c8477a446bdaf4e96a8daff8a2e319499e8/composer.json#L41-L47